package.json

```diff
@@ -64,7 +63,7 @@
     "@vitest/coverage-v8": "1.5.3",
     "@vitest/ui": "1.5.3",
     "algoliasearch": "^4.18.0",
-    "canvas": "^3.1.0",
+    "skia-canvas": "^2.0.2",
```

Comment:
Minor - but this isn't alphabetical (think this is my fault)

Reply:
Fixed